src/transaction/transaction.rs

```diff
@@ -804,10 +889,17 @@ impl TransactionOptions {
             }
         }
     }
+
+    pub fn no_heart_beat(mut self) -> TransactionOptions {
+        self.heartbeat = false;
+        self
+    }
 }
 
 /// The default TTL of a lock in milliseconds
 const DEFAULT_LOCK_TTL: u64 = 3000;
+/// The default heartbeat interval in milliseconds
+const DEFAULT_HEARTBEAT_INTERVAL: Duration = tokio::time::Duration::from_millis(10000);
```

Comment:
```suggestion
const DEFAULT_HEARTBEAT_INTERVAL: Duration = tokio::time::Duration::from_secs(1);
```
Considering `DEFAULT_LOCK_TTL` is 3s, and we don't use/calculate other lock TTLs like in TiDB, I think we can set the interval to 1 second or so.